[-Werror=missing-braces]

When I compiled the firmware, I got the error as following, 

/src/modules/uavcan/uavcan_main.cpp:1263:3: error: missing braces around initializer for 'const char* [4]' [-Werror=missing-braces]
compilation terminated due to -Wfatal-errors.
cc1plus.exe: all warnings being treated as errors
make[3]: *** [src/modules/uavcan/CMakeFiles/modules__uavcan.dir/uavcan_main.cpp.obj] Error 1
make[2]: *** [src/modules/uavcan/CMakeFiles/modules__uavcan.dir/all] Error 2

How can I solve this problem?

--- a/src/modules/uavcan/uavcan_main.cpp
+++ b/src/modules/uavcan/uavcan_main.cpp
@@ -1258,10 +1258,10 @@ UavcanNode::print_info()
 	// Printing all nodes that are online
 	std::printf("Online nodes (Node ID, Health, Mode):\n");
 	_node_status_monitor.forEachNode([](uavcan::NodeID nid, uavcan::NodeStatusMonitor::NodeStatus ns) {
-		static constexpr std::array<const char*, 4> HEALTH {
+		static constexpr std::array<const char*, 4> HEALTH = {
 			"OK", "WARN", "ERR", "CRIT"
 		};
-		static constexpr std::array<const char*, 8> MODES {
+		static constexpr std::array<const char*, 8> MODES = {
 			"OPERAT", "INIT", "MAINT", "SW_UPD", "?", "?", "?", "OFFLN"
 		};
 		std::printf("\t% 3d %-10s %-10s\n", int(nid.get()), HEALTH.at(ns.health), MODES.at(ns.mode));

